localstack/services/dynamodb/provider.py

```diff
@@ -179,15 +180,46 @@
 MANAGED_KMS_KEYS = {}
 
 
+class DynamoDbStreamRecord(TypedDict, total=False):
+    ApproximateCreationDateTime: int
+    SizeBytes: int
+    Keys: Key
+    StreamViewType: Optional[StreamViewType]
+    OldImage: Optional[AttributeMap]
+    NewImage: Optional[AttributeMap]
+    SequenceNumber: Optional[int]
+
+
+class StreamRecord(TypedDict, total=False):
+    """
+    Global record type, this can contain both a KinesisRecord and a DynamoDBStreams record?
+    """
+
+    eventName: str
+    eventID: str
+    eventVersion: str
+    dynamodb: DynamoDbStreamRecord
+    awsRegion: str
+    eventSource: str
+
+
+StreamRecords = list[StreamRecord]
+
+
+class TableRecords(TypedDict):
+    table_stream_type: TableStreamType
+    records: StreamRecords
+
+
+RecordsMap = dict[TableName, TableRecords]
+
+
```

Comment:
I think these should live in `.models`

Reply:
Makes sense, there's the small dependency to the `utils` one, but I've moved it to `models` too. Should make more sense!